modules/serverless-tech-preview-features.adoc

```diff
@@ -13,91 +13,119 @@ The following table provides information about which {ServerlessProductName} fea
 // OCP + OSD table
 ifdef::openshift-enterprise,openshift-dedicated[]
 .Generally Available and Technology Preview features tracker
-[cols="3,1,1",options="header"]
+[cols="3,1,1,1",options="header"]
 |====
-|Feature |1.23|1.24
+|Feature |1.23|1.24|1.25
 
 |`kn func`
 |TP
 |TP
+|GA
```

Comment:
Wondering if we should split lines with what is GA and what is TP. for example:
```
Feature                1.23   1.24   1.25
kn func (Quarkus)       TP     TP     GA
kn func                 TP     TP     TP
```
wdyt? /cc @nainaz

Reply:
@jrangelramos Implemented, but also added the `kn func (other runtimes)` entry. With this added distinction, can we move the `kn func` entry to GA for 1.25 or should it be kept as TP?

Reply:
@rh-max Yes! `kn func` entry could be GA, and I think `kn func invoke` row could be removed from the list since it would be part of `kn func` entry IMO.

Reply:
@jrangelramos @lance Thanks, implemented.